dnf/util.py

```diff
@@ -272,6 +274,21 @@ def touch(path, no_create=False):
         pass
 
 
+def _terminal_messenger(tp='write', msg="", out=sys.stdout):
```

Comment:
Please consider adding option for immediate flushing, so you don't have to call '_terminal_messenger' twice.